Correct spelling error in login

moan0s opened the pull request.

Commit:
Correct spelling error in login

shukon commented:
Sollten wir an dieser Stelle vielleicht direkt die Fehlermeldung informativer gestalten? So dass die Aussage tatsächlich stimmt? :)

Commit:
Rewrite login procedure

The rewriting splits the login process in checking of the username
and then checcking the password of the user. This enbables more
meaningful error messages

moan0s commented:
Ich habe das mal versucht :) Der Loginprozess ist jetzt zweigeteilt siehe commit message

shukon commented on bdp_app/lib/pages/LoginPage.dart:
```diff
@@ -65,18 +65,26 @@ class _LoginPageState extends State<LoginPage> {
+	    barrierDismissible: true,
+	    builder: (BuildContext context) {
+		return AlertDialog(
+		title: new Text("Benutztername falsch"));
```
Da ist ein typo: Benutzername statt Benutz**t**ername

moan0s replied:
Ich habe gedacht das gehört so weil Benutzername generisch maskulin ist

shukon replied:
Achso. Ja ok :)

shukon commented on bdp_app/lib/pages/LoginPage.dart:
```diff
@@ -65,18 +65,26 @@ class _LoginPageState extends State<LoginPage> {
+		barrierDismissible: true,
+		builder: (BuildContext context) {
+		    return AlertDialog(
+		    title: new Text("Psswort falsch"));
```
Wenn schon so speziell, dann richtig: "Passwort", da fehlt ein a :)

moan0s replied:
Oh ja vielen Dank!
